### ISSUE TITLE
HA 0.92 Manifest Creation

### ISSUE THREAD

**Issue by stringy92** [TimSoethout/goodwe-sems-home-assistant#3]
Just upgrading to 0.92 and for custom components you require a Manifest.json for each component - https://developers.home-assistant.io/docs/en/creating_integration_manifest.html

 I have created one which passed the config checks, but then would completely crash HA and wouldnt reboot

**Comment by bouwew** [comment]
It works for me with the following contents:
```{
  "domain": "sems",
  "name": "Goodwe SEMS scraper",
  "documentation": "https://github.com/TimSoethout/goodwe-sems-home-assistant",
  "dependencies": [],
  "codeowners": [],
  "requirements": []
}
```

**Comment by TimSoethout** [comment]
Fixed by #4.
